.circleci/verbatim-sources/job-specs/binary-job-specs.yml

```diff
@@ -324,3 +324,32 @@
           cat "$script"
           source "$script"
 
+  anaconda_prune:
+    parameters:
+      packages:
+        type: string
+        description: "What packages are we pruning? (eg. pytorch, torchvision, torchaudio, etc.)"
+        default: "pytorch"
+      channel:
+        type: string
+        description: "What channel are we pruning? (eq. pytorch-nightly)"
+        default: "pytorch-nightly"
+    docker:
+      - image: continuumio/miniconda3
+    environment:
+      - PACKAGES: "<< parameters.packages >>"
+      - CHANNEL: "<< parameters.channel >>"
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          no_output_timeout: "1h"
+          command: |
+            conda install -yq anaconda-client
+      - run:
+          name: Prune packages
+          no_output_timeout: "1h"
+          command: |
+              ANACONDA_API_TOKEN="${CONDA_PYTORCHBOT_TOKEN}" \
```

Comment:
this might be a dumb question but where is `CONDA_PYTORCHBOT_TOKEN` set?

Reply:
Not a dumb question.

This is set within circleci's credential store under the `org-member` context.